tests/evals/test_dataset.py

```diff
@@ -1496,10 +1496,36 @@ async def mock_async_task(inputs: TaskInput) -> TaskOutput:
                     'assertion_pass_rate': 1.0,
                     'logfire.msg_template': 'evaluate {name}',
                     'logfire.msg': 'evaluate mock_async_task',
+                    'experiment.metadata': {
+                        'n_cases': 2,
+                        'averages': {
```

Comment:
If we're going to be using this on the frontend, I don't know if it's wise to pass along the entire `ReportCaseAggregate` object or just the subset we need